test/analysis/weightFunction.test.ts

```diff
@@ -2,7 +2,7 @@ import 'ts-jest';
 import { buildSchema, DocumentNode, parse } from 'graphql';
 import { TypeWeightObject } from '../../src/@types/buildTypeWeights';
 import buildTypeWeightsFromSchema from '../../src/analysis/buildTypeWeights';
-import ASTParser from '../../src/analysis/ASTParser';
+import ASTParser from '../../src/analysis/QueryParser';
```

Comment:
Might as well rename this from `ASTParser` to `QueryParser` as well

Reply:
Good catches